tls/s2n_x509_validator.h

```diff
@@ -65,9 +65,12 @@ uint8_t s2n_x509_trust_store_has_certs(struct s2n_x509_trust_store *store);
 /** Initializes the trust store to default system paths **/
 int s2n_x509_trust_store_from_system_defaults(struct s2n_x509_trust_store *store);
 
+/** Initialize trust store from a PEM. This will allocate memory, and load PEM into the Trust Store **/
```

Comment:
technically, it will initialize the trust store on the first call, and add the pem on each call right?

Reply:
Yes